pandora-client-web/src/components/header/Header.tsx

```diff
@@ -146,15 +196,69 @@ function FriendsHeaderButton(): ReactElement {
 			iconAlt={ `${ notificationCount } Contacts` }
 			title='Contacts'
 			badge={ notificationCount }
-			onClick={ () => navigate('/relationships') } />
+			onClick={ () => {
+				navigate('/relationships');
+				onClickExtra?.();
+			} }
+		/>
+	);
+}
+
+function OverlayHeader({ onClose: close, visible }: {
+	onClose: () => void;
+	visible: boolean;
+}): ReactElement {
+	return (
+		<DialogInPortal priority={ 5 }>
+			<Column className={ classNames('OverlayHeader', visible ? null : 'hide') }>
+				<Row>
+					<button className='HeaderButton' onClick={ close }>
+						<IconCross />
+					</button>
+				</Row>
+
+				<Column className='content'>
+					<LeftHeader />
+					<hr />
+					<RightHeader onAnyClick={ close } />
+				</Column>
+			</Column>
+		</DialogInPortal>
+	);
+}
+
+function CollapsableHeader(): ReactElement {
+	const currentAccount = useCurrentAccount();
+	const [showMenu, setShowMenu] = useState(false);
+
+	return (
+		<Row alignX='space-between' alignY='center' className='flex-1'>
+			<button className='HeaderButton' onClick={ () => {
+				setShowMenu(true);
+			} }>
+				<IconHamburger />
+			</button>
+			<span>{ currentAccount != null ? currentAccount.username : '[not logged in]' }</span>
```

Comment:
Thinking about it more, I feel it makes not much sense to show the acocunt name here. It is more likely you use different characters of the same account in different tabs, needing a way to quickly see which character the tab is.
Of course you can still show "no character selected" or "not logged in" there, too.

For consistency reasons, I would also recommend to switch the location of the hamburger menu to the right and the character name to the left, which doubles as a character menu, like in the big desktop version.

For the open hamburger menu, the X would then also need to travel to the right side.

Reply:
Got a live demo of my requested changes. I am now happy with it!~